src/Twilio/TwiML/TwiML.cs

```diff
@@ -120,7 +124,18 @@ protected virtual XNode ToXml()
         {
             var elem = new XElement(this.TagName, this.GetElementBody());
 
-            this.GetElementAttributes().ForEach(attr => elem.Add(attr));
+            this.GetElementAttributes().ForEach(attr =>
```

Comment:
Can't we skip a loop by adding to the loop below that starts on line 140?

And is there any way in C# to make the if/else for TryGetValue cleaner? Maybe a ternary?

Reply:
The first loop is for Element attributes that are set with params -  `var elem = new Prompt(1,"expiration-date");` 
The second loop is for options set with SetOption - ` var elem = new Prompt();
            elem.SetOption("for", "expiration-date");`
In second case, the user can use the attribute name whereas in first case we have to transform "for" attribute name.

Reply:
Regarding ternary operation, we could have done it the following way but then end up creating a new XAttribute object every time.
`
string transformedAttr = attr.Name.LocalName;
AttributeNameMapper.TryGetValue(attr.Name.LocalName, out transformedAttr))
elem.Add(new XAttribute(transformedAttr, attr.Value));
'